### ISSUE TITLE
Infinite Duplication Configuration Display

### ISSUE THREAD

**Issue by joelamltc** [directus/directus#14966]
### Describe the Bug

This is more like a user interface problem to me. The Duplication Configuration allows infinite expansion to the next relationship like a loop.

I have a simple Location model setup with Translation. Settings as below:

![Screenshot 2022-08-09 at 15 01 48](https://user-images.githubusercontent.com/13143636/183585196-403deb65-ec59-4979-9a5d-c3e851383a6e.png)

![Screenshot 2022-08-09 at 15 01 55](https://user-images.githubusercontent.com/13143636/183585235-6b76334f-f0fb-453e-856e-3ca6ad2a2644.png)

It results in infinite expansion:

![Screenshot 2022-08-09 at 15 01 04](https://user-images.githubusercontent.com/13143636/183585335-03966860-8c4e-4458-9a61-85abd427e513.png)

### To Reproduce

- Create a model with Translation
- Go Duplication configuration
- Keep clicking expand icon

### Errors Shown

No error shown

### What version of Directus are you using?

v9.15.1

### What version of Node.js are you using?

v14.18.3

### What database are you using?

mysql  Ver 15.1 Distrib 10.5.5-MariaDB, for osx10.15 (x86_64) using readline 5.1

### What browser are you using?

Version 103.0.5060.134 (Official Build) (x86_64)

### How are you deploying Directus?

Running locally

**Comment by br41nslug** [comment]
I don't think this is a bug to be honest. This is a consequence of having a circular relation in the translations interface (or any M2M really) currently this is the expected result if you keep going down the relation loop. 
I'll close this issue for now but I do admit maybe not allowing this recursion could be a nice improvement.
